File: Linguagem C Exercicios/Vetores/NUMEROS PARES.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

   int n,cont;

   printf("Quantos numeros voce vai digitar? ");
   scanf("%d",&n);

   int vet[n];

   // Leitura dos vetores
   for(int i=0;i<n;i++){
    printf("Digite um numero: ");
    scanf("%d",&vet[i]);
   };

   //Numeros pares do vetor
   cont = 0;
   printf("\nNUMEROS PARES: \n");
   for (int i=0;i<n;i++){
    if(vet[i] % 2 == 0){
        printf("%d ",vet[i]);
        cont++;
    };
   };
   printf("\n\nQUANTIDADE DE PARES = %d",cont);



    return 0;
}

File: Linguagem Python/Vetores/NumerosPares.py
n = int(input("Quantos numeros voce vai digitar? "))

vet: [float] = [0 for x in range(n)]

for i in range(0,n):
    vet[i] = float(input("Digite um numero: "))

print()
maiorValor = vet[0]
posMaior = 0
for i in range(1,n):
    if vet[i] > maiorValor:
        maiorValor = vet[i]
        posMaior = i

print(f"MAIOR VALOR = {maiorValor:.1f}")
print(f"POSICAO DO MAIOR VALOR = {posMaior}")